wg-device-management/charter.md

```diff
@@ -0,0 +1,100 @@
+# WG Device Management Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter
+README] and uses the Roles and Organization Management outlined in
+[wg-governance].
+
+## Scope
```

Comment:
We have notes about device support in WG Batch... I would suggest removing it from the charter as part of this PR. But worth bringing up the topic in a WG Batch meeting too.

Reply:
Seems like that could be done in a follow-up PR. I don't want to gate this on that. We have some urgency to move forward.

Reply:
I think we can iterate on that part but it sounds like that's probably a quick-follow-up.

Reply:
There will be overlaps between this WG, batch and serving naturally. I think it will boil down to proper coordination when those overlapping areas are being worked on, to accommodate the input from the other interested parties.

Reply:
Just to update here, serving has updated their charter that  clarified their scope to not overlap with batch.